gc/base/MarkingScheme.cpp

```diff
@@ -336,19 +359,16 @@ MM_MarkingScheme::completeMarking(MM_EnvironmentBase *env)
 void
 MM_MarkingScheme::markLiveObjectsComplete(MM_EnvironmentBase *env)
 {
-	_extensions->collectorLanguageInterface->markingScheme_markLiveObjectsComplete(env);
+	_delegate.markLiveObjectsComplete(env);
 }
 
 MM_WorkPackets *
 MM_MarkingScheme::createWorkPackets(MM_EnvironmentBase *env) {
 	MM_WorkPackets *workPackets = NULL;
 
-#if defined(OMR_GC_MODRON_CONCURRENT_MARK)
-	if(_extensions->concurrentMark) {
+	if (_extensions->isConcurrentMarkEnabled()) {
 		workPackets = MM_WorkPacketsConcurrent::newInstance(env);
```

Comment:
This code should still be wrapped in the #if defined OMR_GC_MODRON_CONCURRENT_MARK since the class itself is probably wrapped in it.  If you compiled without it, it would likely fail to compile.